docker/Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: image
+
+image:
+	docker build . -t isekai
```

Comment:
Please put in some "shards update" support into docs or makefile.

Reply:
Already updated, thanks!